src/ptvsd/server/multiproc.py

```diff
@@ -209,11 +207,8 @@ def notify_root(port):
 
     if not response['incomingConnection']:
         ptvsd.server.log.debug('No IDE connection is expected for this subprocess; unpausing.')
-        debugger = get_global_debugger()
-        while debugger is None:
+        while not ptvsd.is_attached():
```

Comment:
This is not doing the right thing - note the message it logs right above.

The point of that code originally was to basically unblock the `wait_for_attach()` call that is happening on the main thread because we started the subprocess with `--wait`. 

So, it looks like we'll need an API for that also. Basically, `_pydevd.wait_for_attach()` needs to be interruptible somehow. So it'll probably need to use a condition variable there so that it can wait on two events simultaneously - one for `configurationDone`, and one for the interrupt request.